assets/css/base/_base.scss

```diff
@@ -22,3 +24,45 @@ h3 {
 p {
   font-size: 1em;
 }
+
+// Links and Buttons
+a {
+  color: $blue-dark;
```

Comment:
@TeddTech I think this is the wrong colour — looking at Figma I believe the links are actually `#5e82ae` (`$blue-alt`).

Reply:
Okay, I will change I chose `#6895c6` it was the closest one in the primary colours. I must not understand the semantics behind the labels Primary vs Secondary. I would love to go over this come Monday.